llvm/include/llvm/MC/MCObjectStreamer.h

```diff
@@ -81,6 +80,7 @@ class MCObjectStreamer : public MCStreamer {
   bool isIntegratedAssemblerRequired() const override { return true; }
 
   void emitFrames(MCAsmBackend *MAB);
+  MCSymbol *emitCFILabel() override;
```

Comment:
Just to check this was intentional. I don't see any problems with it, but at first glance it doesn't look related.

Reply:
Making the function public is intentional as mentioned by the description:

> emitCFILabel, called by ARMWinCOFFStreamer.cpp, has to be made public.

In the call site, the previous `getParser().getStreamer().emitThumbFunc(Symbol);` somehow was able to access `emitThumbFunc`, despite it being private. Perhaps there was a `friend` somewhere.

The new `getTargetStreamer().emitThumbFunc(Symbol);` does require `emitThumbFunc` to be public.